components/molecules/registration/RegistrationCard.vue

```diff
@@ -24,3 +26,8 @@ const props = defineProps<{
     link: string
 }>()
 </script>
+<style>
+.fs-text {
+    font-size: 1.75rem;
+}
```

Comment:
if you find that you have to make more font sizes, why not just extend bootstrap utility like we're already doing in variables.scss

Reply:
Looking into it